### ISSUE TITLE
Last release was in 2017, will there be a new one?

### ISSUE THREAD

**Issue by rses** [FlexTradeUKLtd/jfixture#61]
Hello

It seems that the release into maven repositories was in 2017. Will there be a new one?

I just noticed that I'm still using my own internal `FixtureExtension` on my private projects and would like to use one published publicly.

Thank you!

**Comment by philipwhiuk** [comment]
Yes, this is long overdue. I'll try to get a release together during next week.

**Comment by tremblaysimon** [comment]
Hello, just a little bump on that issue to know if it would be possible to publish a new version.

**Comment by rses** [comment]
Interesting, @philipwhiuk I'm not sure what the current status of these projects are from Flextrade's perspective, but I'm more than willing to help out with things like releases if you're short on resources.

**Comment by rvplauborg** [comment]
A new release would be great!
